### PR TITLE
Codeword highlighting

### LINKED ISSUE #1466: Codeword highlighting
On TG, codewords/responses in chat are automatically highlighted if you're a traitor. We should have the same system.

### PR DESCRIPTION
**Pull request by aidan-bowman**
## About the Pull Request

Fixes up codeword regeneration + a component that highlights codewords (actually just recolors them because highlighting is too fancy for me).

Closes #1466 

It's not shitcode I SWEAR

## Why It's Good For The Game

muh TG features.

## Changelog

:cl:
add: Traitor code phrases and responses are automatically re-colored!
/:cl:

### CODE REVIEW

**Comment by TheDarkElites** [comment]
This should be split up into 129 PRs, one per each line of code.

**Comment by pumpkintheproot** [comment]
> This should be split up into 129 PRs, one per each line of code.

not doing this is like so anonhault behaviour tho

**Comment by tichys** [comment]
imagine not finishing a pr AND failing checks

**Comment by aidan-bowman** [comment]
lmao span unit tests run on commented-out lines

**Comment by TheDarkElites** [comment]
I sure do hope cheese tested this

**Comment by aidan-bowman** [comment]
> I sure do hope cheese tested this

i actually did this time (you can't make fun of me!!)

File: code/_helpers/names.dm [hunk 1]
```diff
@@ -98,85 +98,68 @@ var/syndicate_name = null
...
+						. += pick(nouns)
 					if(2)
-						code_phrase += pick(GLOB.adjectives)
+						. += pick(GLOB.adjectives)
 					if(3)
-						code_phrase += pick(GLOB.verbs)
-		if(words==1)
-			code_phrase += "."
-		else
-			code_phrase += ", "
```
Comment (Tsurupeta) [inline code/_helpers/names.dm:177]: ?

Comment (aidan-bowman) [reply code/_helpers/names.dm:177]: Previous proc spat out text, now it spits out a list that's converted to text when needed

**Review by Tsurupeta — APPROVED** [review]
whatever

**Pull request merged**